tests/tensor/test_basic.py

```diff
@@ -2386,194 +2386,120 @@ def test_is_flat():
     assert not ptb.is_flat(X.reshape((iscalar(),) * 3))
 
 
-def test_tile():
-    """
-    TODO FIXME: Split this apart and parameterize.  Also, find out why it's
-    unreasonably slow.
-    """
+class TestTile:
+    @pytest.mark.parametrize(
+        "A_shape, reps_test",
+        [
+            ((), (2,)),
+            ((5,), (2,)),
+            ((2, 4), (2, 3)),
+            ((2, 4), (2, 3, 4)),
+            (
+                (2, 4, 3),
+                (
+                    2,
+                    3,
+                ),
+            ),
```

Comment:
Maybe turn formatting off here? `# fmt: off` and `# fmt: on` I believe

Reply:
I just remove the last comma and it will be happy about having them in one line